cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -193,26 +193,26 @@ func (t *templateBuilder) buildNodeFromTemplate(mig *Mig, template *gce.Instance
 	return &node, nil
 }
 
-func (t *templateBuilder) buildNodeFromAutoprovisioningSpec(mig *Mig, cpu int64, mem int64) (*apiv1.Node, error) {
-	if mig.spec == nil {
-		return nil, fmt.Errorf("no spec in mig %s", mig.Name)
+func (t *templateBuilder) buildNodeFromAutoprovisioningSpec(mig Mig, cpu int64, mem int64) (*apiv1.Node, error) {
```

Comment:
It's no longer called 'autoprovisioningSpec'.

Reply:
Done